src/ontology/edits/oeo-physical.omn

```diff
@@ -9262,6 +9262,121 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1359",
         OEO_00140104
     
     
+Class: OEO_00310023
+
+    Annotations: 
+        OEO_00040001 "http://openenergy-platform.org/ontology/oeo/oeo-physical"^^xsd:string,
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A heat recovery steam generator is a heat exchanger that recovers heat from a hot steam.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/1256
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1362",
+        rdfs:label "heat recovery steam generator"
+    
+    SubClassOf: 
+        OEO_00140102,
+        OEO_00000503 some OEO_00110001,
+        OEO_00010234 some OEO_00000007
```

Comment:
Heat goes in, steam comes out.  What has chemical energy (OEO_00000007) to do with that?

Reply:
To axiomatise the relation to `steam`, see my suggestion in the comment above.
Please use `'has energy input' some 'recovered heat'` instead of `uses some recovered heat`.